### PR TITLE
Button component

### PR DESCRIPTION
**Pull request by laila-sb**
Added reusable button component +  css file + added component to Apps.js to test

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 import './App.css';
+import Button from 'react-bootstrap/Button';
```
Comment (mehtapparkinson) [inline frontend/src/App.js:2]: Redundant can be deleted

File: frontend/src/ReusableButton/ReusableButton.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    font-size: 16px;
+    padding: 10px 20px;
+    border-radius: 5px;
+    transition: background-color 0.3s ease, border-color 0.3s ease;
+  }
+  
+  /* Custom hover effect */
+  .button:hover {
+    background-color: #0056b3;
+    border-color: #004085;
```
Comment (mehtapparkinson) [inline frontend/src/ReusableButton/ReusableButton.css:11]: Hover effect chosen colors does not seems to meet accessibility criterias
https://webaim.org/resources/contrastchecker/

File: frontend/src/App.js (hunk 1 above)
Comment (laila-sb) [reply frontend/src/App.js:2]: missed that one

File: frontend/src/ReusableButton/ReusableButton.css (hunk 2 above)
Comment (laila-sb) [reply frontend/src/ReusableButton/ReusableButton.css:11]: Chosen colors are just a starting point, colors for project have not been chosen yet

**Review by mehtapparkinson — APPROVED** [review]
needs updates for the colours in the future but well structured reusable button. great work

**Pull request merged**